### DISCUSSION TITLE
Godot's asset library development

### DISCUSSION THREAD

**Discussion by vaguesyntax** [godotengine/godot-proposals#10640] [General Discussions]
Greetings, I have been using Godot for almost 2 years and it feels great that Godot has developed so well and compared well with the best game engines on the market. Although these matches are generally head-to-head, there is one issue where Godot clearly loses. This is the asset library. Even though it's full inside for now, it looks very simple in appearance, regardless of whether it's web or engine. In other words, although we sometimes cannot find what we are looking for, it is really far behind its competitors in terms of accessibility and appearance. Especially when I look at Unity's asset library, I wish Godot had such an image. That's why I want Godot's asset library to be renewed and improved for reasons such as accessibility and appearance.

![image](https://github.com/user-attachments/assets/c52fa03f-76b3-49a5-9998-85365d9ff491)

**Comment by Meorge** [comment]
I recall hearing that there is a revamped version of the Asset Library in development; not sure exactly how far along it is, but it is certainly something that others are aware of and working on
